languages/julia/generic/Parse_julia_tree_sitter.ml

```diff
@@ -811,20 +811,26 @@ and map_binary_expression (env : env) (x : CST.binary_expression) : expr =
       | ">>>" -> opcall (LSR, v2) [ v1; v3 ]
       | s ->
           Call (N (H2.name_of_id (s, v2)) |> G.e, fb [ Arg v1; Arg v3 ]) |> G.e)
-  | `Exp_times_op_exp (v1, v2, v3) ->
+  | `Exp_times_op_exp (v1, v2, v3) -> (
       let v1 = map_expression env v1 in
       let v2 = (* times_operator *) token env v2 in
       let v3 = map_expression env v3 in
-      opcall (Mult, v2) [ v1; v3 ]
-  | `Exp_choice_tok_choice_dot_choice_plus_exp (v1, v2, v3) ->
+      (* The Julia tree-sitter-grammar groups "&" as times operator *)
+      match Tok.content_of_tok v2 with
+      | "&" -> opcall (BitAnd, v2) [ v1; v3 ]
```

Comment:
Are we sure we handle all the cases? 
Maybe we should a case for | "*" -> ... | _else_ -> failwith "unrecognized operator"

Reply:
I'm conflicted since in principle this should be done correctly by tree-sitter already and i dont want to introduce new parse errors. For correctness sake we could introduce it though. Wdyt?